### PR TITLE
Update foo.py

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

File: src/foo.py [hunk 1]
```diff
@@ -7,3 +7,15 @@ def hello():
     print("Hello foo4")
 
     print("Hello foo5")
+
+
+    print("Hello foo5")
+
+
```
Comment (henry-richard) [inline src/foo.py:14]: See issue one

File: src/foo.py [hunk 2]
```diff
@@ -7,3 +7,15 @@ def hello():
...
+    print("Hello foo5")
+
+
+    print("Hello foo5")
```
Comment (henry-richard) [inline src/foo.py:18]:
```suggestion
    print("Hello foo21")
```

**Review by henry-richard — CHANGES_REQUESTED** [review]

**Review by bert-griffin — APPROVED** [review]

**Review by henry-richard — APPROVED** [review]

**Review by henry-richard — COMMENTED** [review]
Looks good to me

**Review by henry-richard — APPROVED** [review]
Looks okay

**Review by henry-richard — APPROVED** [review]
minor changes

**Pull request merged**